### ISSUE TITLE
Expand JS dict

### ISSUE THREAD

**Issue by heckerfr0d** [heckerfr0d/gitfitti-web#6]
Anyone can contribute. Just head on to [gitfitti](https://gitfitti.herokuapp.com/contribute/) and follow the instructions to help us expand our JS dictionary. PRs are auto-generated on providing Personal Access Token.

Go ahead and  add lowercase, numbers, letters and anything else you think we're missing :innocent:

**Comment by REX51** [comment]
Hi @heckerfr0d 
I added to the list in [gitfitti](https://gitfitti.herokuapp.com/contribute/) with username - access token, but not sure how the PR is being generated. Can you explain it to me?

**Comment by Pavithradevadiga** [comment]
Hey i still see the below erro
![errorpng](https://user-images.githubusercontent.com/34917257/142997318-db51cf41-a54b-422b-8354-d2e1fe5c3695.png)
r

**Comment by heckerfr0d** [comment]
Hii, thank you for your contribution!
It has worked and I have merged your pull request.
Could you tell me what the URL was when you saw this error page?

**Comment by Pavithradevadiga** [comment]
Hi @heckerfr0d , I tried again and this is the link (https://gitfitti.herokuapp.com/contribute) with the error.
![error](https://user-images.githubusercontent.com/34917257/143213687-65d720d5-ce57-4bad-8be0-4c2575ec462a.png)

**Comment by heckerfr0d** [comment]
Hi @Pavithradevadiga 
Please tell me how to reproduce this issue :sweat_smile: 
Like, maybe a list of steps?
I tried making a contribution using another account but I'm not getting this error :")
![image](https://user-images.githubusercontent.com/58948658/143240692-9cce78d7-9457-49b2-924b-cc26066954e7.png)

**Comment by Pavithradevadiga** [comment]
Yeah.

- So I just opened the [gitfitti](https://gitfitti.herokuapp.com/contribute/) link
- Created a simple symbol.
- Named it,added an alias,username and access token
- Clicked on Save
- And BAM i get the error even though the pull request is created :(

I tried reproducing the error again what i observed was I am not facing the error when only  "Translate Text to Graffitti" and "Alias" and "username" are selected, it's  happening when I give personal access token

**Comment by heckerfr0d** [comment]
Hi @Pavithradevadiga 
Thank you for the detailed bug report. We've determined that this weird error only appears when using mobile network and is because the request somehow gets interrupted. As a workaround, we've switched from gunicorn to waitress for serving the web app and the bug has been fixed (hopefully :crossed_fingers:  - please confirm lol). 
Thank you for your help in making our software better :relaxed:

**Comment by Pavithradevadiga** [comment]
Yep,it's resolved .The pull request has been created without any error.

I am Glad I was of help :")(tears of happiness).
